Assets/Scripts/Movement.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+public class Movement : MonoBehaviour
+{
+    public float Speed = 3f;
+    public float AngularSpeed = 360f;
+    void Update()
+    {
+        // movement direction
+        var movement = new Vector3(Input.GetAxis("Horizontal"), 0, Input.GetAxis("Vertical"));
+        transform.position += movement * Speed * Time.deltaTime;
+        
+        if (movement == Vector3.zero)
```

Comment:
floating-point equality checks are not reliable. You should check whether the vector's magnitude is above the threshold